swift-lang/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>io.ecocode</groupId>
         <artifactId>ios-plugin</artifactId>
-        <version>1.1.0-SNAPSHOT</version>
+        <version>1.1.0</version>
```

Comment:
for me, this is the releasing process that will do this king of modifications.
for me, keep "SNAPSHOT" version here and check if releasing process is OK with ios release (or upgrade it)